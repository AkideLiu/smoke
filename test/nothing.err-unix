Missing: TEST-LOCATION...

Usage: smoke-exe [--command ARG] ([-c|--color] | [--no-color]) TEST-LOCATION...
