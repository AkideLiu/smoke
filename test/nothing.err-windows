Missing: TEST-LOCATION...

Usage: smoke-exe.exe [--command ARG] [--bless] ([-c|--color] | [--no-color])
                     [--diff ARG] TEST-LOCATION...
