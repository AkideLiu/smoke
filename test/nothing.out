Usage: smoke [options] TEST-DIRECTORY
    -c, --[no-]color                 Color output
    -h, --help                       Help me
